DPG reference catalog for open standards

```diff
@@ -0,0 +1,54 @@
+###DPG Reference catalog for open standards
+##This reference catalog is an overview of standards that are recommended for the Digital Public Goods. The standards can be technical or semantic, and are sorted by relevant application. 
+
+#Datasets:
+JSON (JavaScript Object Notation)
+XML
+
+#Character encoding and languages:
+UTF-8, ISO/IEC 10646
+ISO 639-1/639-2/639-3
+
+#Web standards:
+HTML 
+CSS
+SSL
```

Comment:
SSL (even SSLv3) is no longer recommended due to security vulnerabilities, and has generally been replaced with TLS v1.2 or v1.3. More information: https://wiki.mozilla.org/Security/Server_Side_TLS